platform_plugin_turnitin/turnitin.py

```diff
@@ -0,0 +1,389 @@
+"""TO-DO: Write a description of what this XBlock is."""
```

Comment:
Can we remove these files related to the Xblock definition? Or are we using them in some way?

Reply:
Done, I removed the xblock related files.